admin_guide/advanced_scheduling/pod_affinity.adoc

```diff
@@ -0,0 +1,351 @@
+[[admin-guide-sched-pod-affinity]]
+= Using Pod Affinity and Anti-Affinity
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+_Pod affinity_ and _pod anti-affinity_ allow you to specify rules about how pods should be placed relative to one another. For example, you could spread or pack pods within a service or relative to pods in other services or, you can make sure a pod is scheduled on a specific type of machine.
```

Comment:
For node affinity:
"Node affinity allows you to constrain which nodes your pod is eligible to be scheduled on using labels on the node. For example, you could configure a pod to only run on a node with a specific CPU or in a specific availability zone."
https://github.com/kubernetes/community/blob/master/contributors/design-proposals/nodeaffinity.md